src/Task.java

```diff
@@ -0,0 +1,28 @@
+public class Task {
+    private final String name;
+    private boolean isComplete;
+
+    public Task(String name, boolean isComplete) {
+        this.name = name;
+        this.isComplete = isComplete;
+    }
+
+    public boolean getIsComplete() {
```

Comment:
Just call it is complete

Reply:
bro it was complete you told me to change it to iscomplete

Reply:
```suggestion
    public boolean isComplete() {
```